keras_model.py

```diff
@@ -0,0 +1,36 @@
+from keras.models import load_model
+from multiprocessing import Lock
+from multiprocessing.managers import BaseManager
+
+import config
+cfg = config.get_localized_config()
+
+class KerasModel():
```

Comment:
Please leave a docstring in this class explaining what it is and linking to https://github.com/Dref360/tuto_keras_web